wiki/osu!academy/en.md

```diff
@@ -52,6 +52,6 @@ If you have any questions or comments, please forward them to [ztrot](https://os
 
 ## External Links
 
-- Source: <https://news.ppy.sh/post/70504430082/introducing-the-osu-academy>
+- Source: <https://osu.ppy.sh/home/news/2013-12-19-introducing-the-osu-academy>
 - Official Youtube channel: <https://www.youtube.com/user/osuacademy/videos>
 - Official thread: <https://osu.ppy.sh/forum/t/169707>
```

Comment:
Update the link with the new site format, so no redirects.

Reply:
> so no redirects

I'm not sure about the redirect mess at this time and what the final links will look like, but the site you get is what you picked from the site switcher thing, so someone will always get redirected.